Components/Communication/Inc/UARTDriver.hpp

```diff
@@ -9,80 +9,70 @@
 */
 
 /* Includes ------------------------------------------------------------------*/
-#include "stm32f4xx_ll_usart.h"
+#include "cmsis_os.h"
 #include "stm32f4xx_hal_rcc.h"
 #include "stm32f4xx_ll_dma.h"
-#include "cmsis_os.h"
+#include "stm32f4xx_ll_usart.h"
 
 /* UART Driver Instances ------------------------------------------------------------------*/
 class UARTDriver;
 
 namespace Driver {
-	extern UARTDriver uart1;
-	extern UARTDriver uart2;
-	extern UARTDriver uart3;
-	extern UARTDriver uart5;
-}
+extern UARTDriver uart1;
+extern UARTDriver uart2;
+extern UARTDriver uart3;
+extern UARTDriver uart5;
+}  // namespace Driver
 
 /* UART Driver Aliases ------------------------------------------------------------------*/
 namespace UART {
-	constexpr UARTDriver* Umbilical_RCU = &Driver::uart1;
-	constexpr UARTDriver* Radio = &Driver::uart2;
-	constexpr UARTDriver* Conduit_PBB = &Driver::uart3;
-	// UART 4 (GPS) uses HAL
-	constexpr UARTDriver* Debug = &Driver::uart5;
-}
+constexpr UARTDriver* Umbilical_RCU = &Driver::uart1;
+constexpr UARTDriver* Radio = &Driver::uart2;
+constexpr UARTDriver* Conduit_PBB = &Driver::uart3;
+// UART 4 (GPS) uses HAL
+constexpr UARTDriver* Debug = &Driver::uart5;
+}  // namespace UART
 
 /* UART Receiver Base Class ------------------------------------------------------------------*/
 /**
  * @brief Any classes that are expected to receive using a UART driver
  *		  must derive from this base class and provide an implementation
  *		  for InterruptRxData
  */
-class UARTReceiverBase
-{
-public:
-	virtual void InterruptRxData(uint8_t errors) = 0;
+class UARTReceiverBase {
+   public:
+    virtual void InterruptRxData(uint8_t errors) = 0;
 };
 
-
 /* UART Driver Class ------------------------------------------------------------------*/
 /**
  * @brief This is a basic UART driver designed for Interrupt Rx and Polling Tx
  *	      based on the STM32 LL Library
  */
-class UARTDriver
-{
-public:
-	UARTDriver(USART_TypeDef* uartInstance) :
-		kUart_(uartInstance),
-		rxCharBuf_(nullptr),
-		rxReceiver_(nullptr) {}
+class UARTDriver {
+   public:
```

Comment:
Would be nice to left-align `public:` `protected:` and `private:`

Reply:
Disagree :)